<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelAttackName">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose an Attack</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAnimalName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select an Animal</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTargetAC">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Target AC</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAnimalNum">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>How Many?</string>
    </property>
   </widget>
   <widget class="QComboBox" name="inputAttackName">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAttack">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>260</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Attack</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputAnimalInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>371</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLoadPack">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Pack</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="inputAnimalNum">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="inputTargetAC">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>190</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="inputAnimalName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="inputDisadvantage">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Disadvantage</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="inputAdvantage">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Advantage</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputAttack">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>290</y>
      <width>261</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="inputMightySummoner">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Mighty Summoner</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDamage">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>260</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Damage</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="inputDamage">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>260</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="outputHitPoints">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>290</y>
      <width>261</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonHeal">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>220</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Heal</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="inputDamageAOE">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>260</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>AOE</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="inputHealAOE">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>220</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>AOE</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="inputHeal">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>220</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="dark_orange/style.qrc"/>
 </resources>
 <connections/>
</ui>
